packages/summer-earn-protocol-gov-subgraph/src/mappings/timelock.ts

```
@@ -61,6 +73,41 @@ export function getOrCreateCrossChainProposal(proposalId: string): CrossChainPro
+ */
+export function getTopic0(eventSig: string): ByteArray {
+  const signature = ByteArray.fromUTF8(eventSig)
+
+  return crypto.keccak256(signature)
+}
+
+export class EventSignature {
+  // EARN
+  static ProposalReceivedCrossChain: string = 'ProposalReceivedCrossChain(uint256,uint32)'
+  static CallSalt: string = 'CallSalt(bytes32,bytes32)'
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)